include/openssl/ssl.h.in

```diff
@@ -1848,6 +1851,7 @@ size_t SSL_client_hello_get0_ciphers(SSL *s, const unsigned char **out);
 size_t SSL_client_hello_get0_compression_methods(SSL *s,
                                                  const unsigned char **out);
 int SSL_client_hello_get1_extensions_present(SSL *s, int **out, size_t *outlen);
+int SSL_client_hello_get_raw_extensions(SSL *s, unsigned short *exts, size_t *num_exts);
```

Comment:
This line is too long and needs to be wrapped.
The continuation line should start at the column after the open parenthesis.